source/crud/crud.md

```diff
@@ -2474,6 +2677,8 @@ aforementioned allowance in the SemVer spec.
 
 ## Changelog
 
+- 2024-10-29: Defined `findOne` operation as optional.
```

Comment:
"as optional" is tentative here pending an outcome of a poll across drivers. I'm considering making the operation required, but letting drivers choose their timeline for implementing this operation, as `findOne` is a common use case.